charts/openebs/templates/_helpers.tpl

```diff
@@ -41,3 +41,126 @@ Create the name of the service account to use
     {{ default "default" .Values.serviceAccount.name }}
 {{- end -}}
 {{- end -}}
+
+
+{{/*
+Define meta labels for openebs components
+*/}}
+{{- define "openebs.common.metaLabels" -}}
+chart: {{ template "openebs.chart" . }}
+heritage: {{ .Release.Service }}
+openebs.io/version: {{ .Values.release.version | quote }}
+{{- end -}}
+
+
+{{- define "openebs.cluster-exporter.name" -}}
```

Comment:
n_: change from `cluster-exporter` to `ndm-cluster-exporter`

Reply:
done